src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/other-condition-types/create-anomaly-alert-conditions.mdx

```diff
@@ -26,92 +27,92 @@ import baselineOneFacet from 'images/baseline-one-facet.png'
+To create an anomaly condition, go to **[one.newrelic.com](https://one.newrelic.com) > AI & Alerts > Policies > (create or select policy) > Create alert condition**: When you start to create a [condition](/docs/alerts/new-relic-alerts/defining-conditions/define-alert-conditions#create-condition), choose one of the following [data sources](#availability):
 
-* NRQL: Create a NRQL condition and then choose baseline
-* APM: Application metric baseline
-* Browser: Metric baseline
+* NRQL: Create a NRQL condition and then choose anomaly
+* APM: Application metric anomaly
+* Browser: Metric anomaly
 
-Here are some tips for setting baseline thresholds:
+Here are some tips for setting anomaly thresholds:
 
-* Set the [baseline direction](#upper-lower) to monitor violations that happen either above or below the baseline.
+* Set the [anomaly direction](#upper-lower) to monitor violations that happen either above or below the anomaly.
 * Set the preview chart to either [2 days or 7 days](#days) of displayed data. (Not applicable for [NRQL alert conditions](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries).)
-* Use the slider bar to adjust the <img title="Alerts v3: Critical alert icon" alt="icon-alert-critical.png" src={iconAlertCritical}/> **Critical** threshold sensitivity, represented in the preview chart by the light gray area around the baseline. The tighter the band around the baseline, the more sensitive it is and the more violations it will generate.
-* Optional: You can create a <img title="Alerts v3: Warning alert icon" alt="icon-alert-warning.png" src={iconAlertWarning}/> [**Warning** threshold](/docs/alerts/new-relic-alerts/defining-conditions/set-thresholds-alert-condition#threshold-levels) (the darker gray area around the baseline).
+* Use the slider bar to adjust the <img title="Alerts v3: Critical alert icon" alt="icon-alert-critical.png" src={iconAlertCritical}/> **Critical** threshold sensitivity, represented in the preview chart by the light gray area around the anomaly. The tighter the band around the anomaly, the more sensitive it is and the more violations it will generate.
+* Optional: You can create a <img title="Alerts v3: Warning alert icon" alt="icon-alert-warning.png" src={iconAlertWarning}/> [**Warning** threshold](/docs/alerts/new-relic-alerts/defining-conditions/set-thresholds-alert-condition#threshold-levels) (the darker gray area around the anomaly).
 * For NRQL alerts, see the [allowed types of NRQL queries](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries#threshold-types).
 * If the alert condition applies to multiple apps, you can select a choice from the dropdown above the chart to use different metrics. (Not applicable for NRQL alert conditions.)
 
-## Faceted baseline conditions [#baseline-facets]
+## Faceted anomaly conditions [#anomaly-facets]
```

Comment:
Double check that all these links are addressed with this change - https://github.com/newrelic/docs-website/search?q=%23baseline-facets

Reply:
fixed!